core/src/test/java/org/apache/calcite/test/RelOptRulesTest.java

```diff
@@ -139,6 +141,15 @@ protected DiffRepository getDiffRepos() {
     return DiffRepository.lookup(RelOptRulesTest.class);
   }
 
```

Comment:
don't add tests at the top of the file (or at the bottom). they cause merge conflicts. put them in some "logical" place.